inspirehep/dojson/hep/fields/bd90x99x.py

```diff
@@ -54,6 +57,11 @@ def get_value(value):
                 year = int(value.get('y'))
             except:
                 pass
+        if 'i' in value:
+            try:
+                isbn = normalize_isbn(value.get('i'))
+            except:
```

Comment:
Yeah, I noticed this is the style of the previous cases but it isn't necessarily the best one :smile: Check [this article](http://stackoverflow.com/questions/21553327/why-is-except-pass-a-bad-programming-practice?answertab=votes#tab-top).